content/docs/user-guide/components/reference/physx/collider.md

```diff
@@ -1,241 +1,142 @@
-
-****Sphere shape****
-
-![\[PhysX Collider component interface, Sphere.\]](/images/user-guide/component/physx/physx/ui-physx-collider-A-1.png)
-
-**Radius**
-Radius multiplier of the sphere collider. The size of the sphere primitive is the **Radius** multiplied by the largest value in the **Scale** property in the **[Transform](/docs/user-guide/components/reference/transform/)** component.
-
-****Box shape****
-
-![\[PhysX Collider component interface, Box.\]](/images/user-guide/component/physx/physx/ui-physx-collider-A-2.png)
-
-**Dimensions**
-Width, depth, and height of the box collider.
-
-****Capsule shape****
-
-![\[PhysX Collider component interface, Capsule.\]](/images/user-guide/component/physx/physx/ui-physx-collider-A-3.png)
-
-**Height**
-Height of the capsule primitive shape. The height property of the capsule must be at least twice the radius property. For example, if the radius of the capsule is **5.0**, the minimum height is **10.0**.
-
-**Radius**
-Radius of the capsule primitive shape. The radius property of the capsule must be no greater than half the height property. For example, if the height of the capsule is **10.0**, the maximum radius is **5.0**.
-
-****Draw Collider****
-Render the collider in the viewport. Enabled by default.
-
-****Edit****
-Enable collider component mode to edit properties of the collider in the viewport using manipulators.
-
-## Static PhysX entities 
-
-A PhysX entity that is static can interact with other entities, but doesn't move.
-
-**To create a static PhysX entity**
-
-1. Create an entity. For more information, see [Creating an Entity](/docs/userguide/creating-entity.md).
-
-1. In the **Entity Inspector**, choose **Add Component** and then select **[Mesh](/docs/userguide/components/static-mesh.md)** component.
-
-1. In the **Mesh** component, choose a mesh asset for the **Mesh asset** property.
-
-1. In the **Entity Inspector**, choose **Add Component** and then select **PhysX Collider** component.
+| Property | Description | Value | Default |
+| - | - | - | - |
+| **PhysX Mesh** | Assign a physics asset to the collider. For more information, refer to [FBX Settings PhysX export](/docs/user-guide/assets/fbx-settings/physx-export/). | PhysX mesh |  |
+| **Asset Scale** | Scale the collider shape independent of the entity. | Vector Scale | X:`1.0`, Y:`1.0`, Z:`1.0` |
+| **Physics Materials from Asset** | The physics materials for this collider are automatically set based on the Physics Materials from the mesh's PhysX asset (refer to [FBX Settings PhysX tab](/docs/user-guide/assets/fbx-settings/settings-physx-tab/)). If the physics material doesn't exist in the **Physics Materials - Library**, the default physics material from will be used. Physics material assignments cannot be edited while this option is on. | Boolean | `On`|
```

Comment:
I removed a missing word. Did you instead want to specify where the default physics material is from? 

```suggestion
| **Physics Materials from Asset** | The physics materials for this collider are automatically set based on the Physics Materials from the mesh's PhysX asset (refer to [FBX Settings PhysX tab](/docs/user-guide/assets/fbx-settings/settings-physx-tab/)). If the physics material doesn't exist in the **Physics Materials - Library**, the default physics material will be used. Physics material assignments cannot be edited while this option is on. | Boolean | `On`|
```